data/links.yaml

```diff
@@ -1286,6 +1286,11 @@ links:
     url: "http://aboutus.ft.com/company/our-standards/editorial-code"
     submenu:
 
+  - &community_events
```

Comment:
I think actually the url for this should be absolute. My mistake. https://www.ft.com/tour/community